### ISSUE TITLE
Include all required dependencies for tests

### ISSUE THREAD

**Issue by nuest** [karolyiar/IlwisJavaExtension#4]
After switching to Maven (#2), please add all required resources to `src/test/resources` and resolve them from there (e.g. using Guava's `Resources` class). Remove the hard coded paths in https://github.com/karolyiar/IlwisJavaExtension/blob/master/ilwisobject_test/test/ilwisobject_test/TestUtil.java#L13

**Comment by karolyiar** [comment]
The compiled Ilwis and the tests should also be in the resources? These
take up ~1.5GB.

2015-07-03 10:11 GMT+02:00 Daniel Nüst notifications@github.com:

**Comment by nuest** [comment]
No, of course not :-). I guess Ilwis is not as easy to test Java applications... **would it be possible to download the required Ilwis binaries from a release, extract and use them?** This download and extract could be automated with a Maven ant task...

The problem might be that your tests do not work yet with any Ilwis release.
